src/layouts/ReviewRequest/components/RequestOverview/RequestOverview.tsx

```diff
@@ -165,6 +167,36 @@ export const RequestOverview = ({
   items,
   allowEditing,
 }: RequestOverviewProps): JSX.Element => {
+  const [fileName, setFileName] = useState("")
```

Comment:
Curious: Any reason for using state for the file name rather than relying on query params? I was thinking if it might be better if we allow users to use the URL to directly enter the diff view of a specific file.

Note: I might be confused with the flow so will need some clarifications!

Reply:
i might do the query params in a follow up PR to avoid bloating this PR - is that ok w/ you @dcshzj

Reply:
Yup that can be separate!